react/features/base/environment/utils.js

```diff
@@ -11,6 +11,15 @@ export function isMobileBrowser() {
     return Platform.OS === 'android' || Platform.OS === 'ios';
 }
 
+/**
+ * Returns whether or not the current environment is an Android mobile device.
+ *
+ * @returns {boolean}
+ */
+export function isAndroidDevice() {
+    return Platform.OS === 'android';
```

Comment:
you use 'android' more than one time, so please export this string into a constant